rules/downgrade-php71/tests/Rector/TryCatch/DowngradePipeToMultiCatchExceptionRector/DowngradePipeToMultiCatchExceptionRectorTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DowngradePhp71\Tests\Rector\TryCatch\DowngradePipeToMultiCatchExceptionRector;
+
+use Iterator;
+use Rector\DowngradePhp71\Rector\TryCatch\DowngradePipeToMultiCatchExceptionRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class DowngradePipeToMultiCatchExceptionRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
```

Comment:
Downgrade rules tests can run only on PHP they're downgrading and hire. So there is no syntax error:

```suggestion
    /**
     * @requires PHP 8.0
     * @dataProvider provideData()
     */
```

Reply:
The rest look good, so I'm merge this :+1: